src/PageContainer/component.js

```diff
@@ -100,8 +100,10 @@ export default class PageContainer extends Component {
     }
 
     const item = find(context.collection, pageUrl)
-
+    console.log(isClient)
+    console.log(typeof window !== "undefined")
```

Comment:
We only use browserHistory in PageContainer so no need for context. So browserHistory, isClient, ... will be passed via props